node/src/components/sync_leaper.rs

```diff
@@ -263,6 +263,7 @@ impl SyncLeaper {
     where
         REv: From<FetcherRequest<SyncLeap>> + Send,
     {
+        info!(%sync_leap_identifier, "Registering leap attempt");
```

Comment:
```suggestion
        info!(%sync_leap_identifier, "registering leap attempt");
```

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/3507/commits/8352a002fa1f86ed2a3ae82ec8440c3a00961e4e